src/bot/integrations/discord.ts

```diff
@@ -98,6 +98,28 @@ class Discord extends Integration {
   })
   sendGeneralAnnounceToChannel = '';
 
+  @settings('bot')
+  @ui({
+    type: 'selector', values: ['online', 'idle', 'invisible', 'dnd'],
+    if: () => self.guild.length > 0,
+  })
+  onlinePresence: 'online' | 'idle' | 'invisible' | 'dnd' = 'online';
+
+  /*
+  @settings('bot')
+  @ui({
+    type: 'toggle-enable',
+    if: () => self.guild.length > 0,
+  })
+  togglePresenceOnStream = true;
+ */
+  @settings('bot')
+  @ui({
+    type: 'selector', values: ['streaming', 'online', 'idle', 'invisible', 'dnd'],
+    if: () => self.guild.length > 0,
+  })
+  togglePresenceOnStream: 'streaming' | 'online' | 'idle' | 'invisible' | 'dnd' = 'online';
```

Comment:
are you sure this is doing exactly what you want? I mean `toggle` should be true/false. Maybe change naming convention.